ccinput/packages/nwchem.py

```diff
@@ -211,7 +199,8 @@ def close_blocks(self):
             self.method_block += " end \n"
         if self.calculation_block != "":
             self.calculation_block += " end \n"
-
+    def to_lowercase(self,text):
+        text.lower()
```

Comment:
Why not use `text.lower()` directly instead of this method? Also, there is a `return` missing in the method.

Reply:
I don't know how that got there, I never actually use that function in the code :D